package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -209,3 +218,155 @@ def __init__(self, strand1: List[Residue], strand2: List[Residue],
             sel2.append(s[1].atoms.select_atoms(f'name {a2}'))
 
         super(WatsonCrickDist, self).__init__(sel1, sel2, **kwargs)
+
+
+class MinorPairDist(NucPairDist):
+    r"""Minor-Pair basepair distance for selected residues over a trajectory.
+
+    Takes two lists of :class:`~MDAnalysis.core.groups.Residue` objects and
+    calculates the Watson-Crick distance between them over the trajectory.
+    Bases are matched by their index in the lists given as arguments.
+
+    Parameters
+    __________
+    strand1: List[Residue]
+        First list of bases
+    strand2: List[Residue]
+        Second list of bases
+    o2_name: str (optional)
+        Name of Oxygen 2 of nucleic acids
+        by default assigned to N1
+    c2_name: str (optional)
+        Name of Carbon 2 of nucleic acids
+        by default assigned to N3
+    g_name: str (optional)
+        Name of Guanine in topology
+        by default assigned to G
+    a_name: str (optional)
+        Name of Adenine in topology
+        by default assigned to G
+    u_name: str (optional)
+        Name of Uracil in topology
+        by default assigned to U
+    t_name: str (optional)
+        Name of Thymine in topology
+        by default assigned to T
+    c_name: str (optional)
+        Name of Cytosine in topology
+        by default assigned to C
+    **kwargs: dict
+        arguments for :class:`~MDAnalysis.analysis.base.AnalysisBase`
+
+    Attributes
+    __________
+    results: numpy.ndarray
+    first index is selection second index is time
+    results.times: numpy.ndarray
+    times used in analysis
+
+    Raises
+    ______
+    ValueError
+    if the residues given are not amino acids
+    ValueError
+    if the selections given are not the same length
+
+    """
+
+    def __init__(self, strand1: List[Residue], strand2: List[Residue],
```

Comment:
`Union(List[Residue], ResidueGroup)` ?

Reply:
I'll add that, I looked at `ResidueGroup` and there is no reason `NucPairDist` can't accept it or `Residue`

Reply:
I used a type alias to do it so that the code is less verbose, what do you think about that solution? It is a bit less clear in the docs, but simplifies the code.

Reply:
I don't understand what a type alias is and where you used it — sorry, apologies for my ignorance. Can you please explain and comment on the corresponding piece of code?

Reply:
At least from the docs it is not clear to me that the class would now also accept `strand1: ResidueGroup` — ... at least from your comment I surmise that's what _should_ be possible now.

Reply:
@orbeckst A type alias is when you give a specific name to a composite type in my case `ResidueClass` is an alias for `Union[Residue, ResidueGroup]` so my code's type signatures are a bit less verbose